pipelines/SpeciesTreeFromBusco/main.nf

```diff
@@ -260,9 +297,10 @@ process runIqtree {
 workflow {
     println(ensemblLogo())
     prepareBusco(params.busco_proteins)
-    genomes = Channel.fromPath("${params.dir}/*.fas")
+    genomes = Channel.fromPath("${params.dir}/*.*")
+    prepareGenome(genomes)
```

Comment:
Is there any possibility that the `params.dir` directory could contain both the compressed and uncompressed genome files? Thinking about a checking step to see if a genome file has already been processed and already exists in the `processed/` dir or skipping if it already exists?

Reply:
I think it should also work in that case as it will only decompress files with ".gz" extension. If that is not present it will just create a symlink.

Reply:
I think the possibility of having both gzipped and non-gzipped files in the input directory is not that high. But from nextflow's perspective, they would show up as different files so they would create a duplication of species which would probably cause the pipeline to fail downstream.